### DISCUSSION TITLE
底部代码无法正常自动复合

### DISCUSSION THREAD

**Discussion by jing5460** [AlistGo/alist#183] [Q&A] [answered]
如图，怎么调式都只是固定在一个位置，手机上也如此，文件夹或文件不超过就正常
![image](https://user-images.githubusercontent.com/38358681/144448650-854f7d69-2862-497b-bdd8-2de15a849ad8.png)
![image](https://user-images.githubusercontent.com/38358681/144449516-65da13fd-8090-4436-b82f-68c709c08544.png)

**Comment by xhofe** [comment, marked as answer]
底部已经包含了script标签。

**Reply by jing5460** [reply]
试过了，行不通

**Reply by jing5460** [reply]
![image](https://user-images.githubusercontent.com/38358681/144469367-9971cf98-8747-403d-b433-41c66bb2cdf6.png)

**Reply by xhofe** [reply]
我看了你加的代码问题很多，本身就是不闭合的 但主要是body的高度默认设置了100%；你可以在自定义样式中加入
```css
html,body {
    height:unset !important;
}
```
重设高度。

**Reply by jing5460** [reply]
谢谢！可以了，  现在还有问题，我应该如何设置居中呢

**Reply by xhofe** [reply]
自己写css。

**Reply by jing5460** [reply]
帮写一个呗
